AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts

```diff
@@ -10,13 +10,21 @@ import { IdentityDetailsComponent } from "./components/quotas/identity/identity-
 import { ClientEditComponent } from "./components/client/client-edit/client-edit.component";
 import { AuthGuard } from "./shared/auth-guard/auth-guard.guard";
 import { LoginComponent } from "./components/shared/login/login.component";
+import { CustomRouteReuseStrategy } from "./utils/custom-route-reuse-strategy";
 
 const routes: Routes = [
     { path: "", redirectTo: "/dashboard", pathMatch: "full" },
     { path: "login", component: LoginComponent },
     { path: "dashboard", component: DashboardComponent, canActivate: [AuthGuard] },
     { path: "identities", component: IdentityListComponent, canActivate: [AuthGuard] },
-    { path: "identities/:address", component: IdentityDetailsComponent, canActivate: [AuthGuard] },
+    {
+        path: "identities/:address",
+        component: IdentityDetailsComponent,
+        canActivate: [AuthGuard],
+        data: {
+            reuseComponent: true
+        }
```

Comment:
I'm not familiar with this. I looked it up and I understand the idea, but I'm having a hard time understanding why it's used here. Could you explain?

Reply:
When we want to link components we generally use something like this:
`await this.router.navigate(["/identities/${identityAddress}"]);`
Which for the most part is fine, but when we want to navigate inside the same component, but with different parameters, using the default router.navigate logic, the current page status is not reset.

We could manually reset the page and run all the methods again, but it becomes tricky when we are also dealing with other imbedded components (Relationships, Messages, etc..), so I figured the best way to handle this is to just make this particular page reload when we navigate with different parameters.

Reply:
Got it, thanks!